groovy-language-server/src/main/java/com/palantir/groovylanguageserver/GroovycWrapper.java

```diff
@@ -116,6 +130,48 @@ public Path getWorkspaceRoot() {
     }
 
     @Override
+    public Map<String, Set<SymbolInformation>> getReferences() {
+        return references;
+    }
+
+    @Override
+    public Set<SymbolInformation> findReferences(ReferenceParams params) {
+        Set<SymbolInformation> symbols = fileSymbols.get(params.getTextDocument().getUri());
+        if (symbols == null) {
+            return Sets.newHashSet();
+        }
+        Set<SymbolInformation> foundSymbolInformations = symbols.stream()
```

Comment:
Yep! We are trying to find the symbol that intersects the given position, so basically the symbol that the user clicked on.